ansible_collections/arista/avd/docs/stylesheets/extra.material.css

```diff
@@ -1,199 +1,219 @@
+    --md-primary-fg-color:               #27569B;
+    --md-primary-fg-color--light:        #FFFFFF;
+    --md-primary-fg-color--dark:         #27569B;
+    --md-primary-bg-color:               #FFFFFF;
+    --md-primary-bg-color--light:        #FFFFFF;
+    --md-default-bg-color: #2e363e;
+
+    /* Accent color shades */
+    --md-accent-fg-color:                #27569B;
+    --md-accent-bg-color:                #27569B;
+    --md-accent-bg-color--light:         #27569B;
+
+    /* Link color */
+    --md-typeset-a-color:                #4473a9;
+    --md-typeset-a-color-fg:             #FFFFFF;
+    --md-typeset-a-color-bg:             #27569B;
+
+    /* background-color: #2e363e; */
+
+    /* Code block color shades */
+    /* --md-code-bg-color:                  #E6E6E6;
+    --md-code-border-color:              #aec6db4f;
+    --block-code-bg-color:               #e4e4e4; */
+  }
+
+  [data-md-color-scheme="default"] {
+    /* Primary color shades */
+    --md-primary-fg-color:               #27569B;
+    --md-primary-fg-color--light:        #FFFFFF;
+    --md-primary-fg-color--dark:         #27569B;
+    --md-primary-bg-color:               #FFFFFF;
+    --md-primary-bg-color--light:        #FFFFFF;
+
+    /* Accent color shades */
+    --md-accent-fg-color:                #27569B;
+    --md-accent-bg-color:                #27569B;
+    --md-accent-bg-color--light:         #27569B;
+
+    /* Link color */
+    --md-typeset-a-color:                #4473a9;
+    --md-typeset-a-color-fg:             #FFFFFF;
+    --md-typeset-a-color-bg:             #27569B;
+
+    /* Code block color shades */
+    --md-code-bg-color:                  #E6E6E6;
+    --md-code-border-color:              #0000004f;
+    --block-code-bg-color:               #e4e4e4;
+
+   /* Code block color shades-v1AN */
+   /*--md-code-bg-color:                  #FFFFFF;
+   --md-code-border-color:              #ffffff;
+
+   /*--block-code-bg-color:               #ffffff;
+
+  }
+
+  @media only screen and (min-width: 76.25em) {
+    .md-main__inner {
+      max-width: 85%;
+    }
+    .md-sidebar--primary {
+      left: 5%;
+    }
+    .md-sidebar--secondary {
+      right: 5%;
+      margin-left: 0;
+      -webkit-transform: none;
+      transform: none;
+    }
+  }
+
+  @media only screen {
+    .md-typeset a:hover {
+      background-color: var(--md-typeset-a-color-bg);
+      color: var(--md-typeset-a-color-fg);
+    }
+    .md-footer-nav {
+        background-color: var(--md-default-bg-color--light);
+        color: var(--md-accent-fg-color--transparent)
+    }
+    .md-footer {
+      height: 2%;
+    }
+    .md-footer-nav__direction {
+      position: absolute;
+      right: 0;
+      left: 0;
+      margin-top: -1rem;
+      padding: 0 1rem;
+      color: var(--md-default-fg-color--light);
+      font-size: .64rem;
+    }
+    .md-footer-nav__title {
+      font-size: .9rem;
+      line-height: 10rem;
+      color: var(--md-default-fg-color--light);
+    }
+
```

Comment:
do we need to keep all the comments?

Reply:
Not at all, they are remnants of history. I'll remove them now

Reply:
This was a mistake on my end, the old comments when removed made these look like comments as well. These are related to how the pane will expand when more real estate is available.